README.md

```diff
@@ -0,0 +1,72 @@
+# webmaker-core
+
+`webmaker-core` is the React based core for the Webmaker app. It's a series of webviews that are integrated into the various platforms running Webmaker (currently: Android, eventually: iOS) and also the native web version of Webmaker (`webmaker-browser`).
```

Comment:
Might even be better (more clear) to just list the platforms instead of listing them inside the intro.
- [Android](https://github.com/mozilla/webmaker-android)
- [Browser](https://github.com/mozilla/webmaker-browser)